charts/minecraft/values.yaml

```diff
@@ -364,6 +364,8 @@ minecraftServer:
     # loadBalancerSourceRanges: []
     ## Set the externalTrafficPolicy in the Service to either Cluster or Local
     # externalTrafficPolicy: Cluster
+    ## set this to false to not have colorized logs
+    tty: true
```

Comment:
Can you move it up a level since it doesn't relate to rcon?

Reply:
Sure, done